packages/platform/atoms/event-types/wrappers/EventAdvancedWebWrapper.tsx

```diff
@@ -0,0 +1,20 @@
+import { EventAdvancedTab } from "@calcom/features/eventtypes/components/tabs/advanced/EventAdvancedTab";
+import type { EventTypeSetupProps } from "@calcom/features/eventtypes/lib/types";
+import { trpc } from "@calcom/trpc/react";
+
+export type EventAdvancedWebWrapperProps = Pick<EventTypeSetupProps, "eventType" | "team">;
+
+const EventAdvancedWebWrapper = (props: EventAdvancedWebWrapperProps) => {
+  const connectedCalendarsQuery = trpc.viewer.connectedCalendars.useQuery();
+  const { data: user, isPending } = trpc.viewer.me.useQuery();
+  return (
+    <EventAdvancedTab
+      {...props}
+      connectedCalendars={connectedCalendarsQuery.data}
```

Comment:
let's rename this props to calendarsQueryData

Reply:
https://github.com/calcom/cal.com/pull/16711/commits/f28430e29737e02eb9c144e4910b257c69c750ef